### ISSUE TITLE
How to implement postprocessing function with DPU?

### ISSUE THREAD

**Issue by acg93-pixel** [Xilinx/Vitis-AI#1381]
Hi, I'm trying to deploy [RTMDet-Ins](https://github.com/open-mmlab/mmdetection/blob/main/mmdet/models/dense_heads/rtmdet_ins_head.py) model from MMdetection framework to VEK280 target. Quantized model performs well, but there is NMS and a convolution in post-processing function (inside _mask_predict_by_feat_single) which is not part of model's forward method. Piece of this method which contains conv:
`for i, (weight, bias) in enumerate(zip(weights, biases)):
       x = F.conv2d(
            x, weight, bias=bias, stride=1, padding=0, groups=num_inst)
       if i < n_layers - 1:
            x = F.relu(x)`

In order to efficiently implement this post-processing method, is there a way to implement this with DPU?

**Comment by lishixlnx** [comment]
No. You have to implement it in postprocess function (either in c++ or in python). You can find some examples in Vitis-Ai-Library.
